mabl snippets/ValueFromCommaSeparatedList.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * TITLE: Value from comma separated list
+ * NOTE: This is a 1-indexed list. If you want the first value from the list, set "index" to "1"
+ * 
+ * @mablParam: list - A comma separated list of values that will work like an array
+ * 
+ * @mablParam: index - The index of the item you would like to get from the list
+ * 
+ * @mablReturn: The value from the index in the list
+ * 
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+    // Get the value of the mabl variable "list"
+    let list = mablInputs.variables.user.list;
```

Comment:
are there cases where mablInputs, variables, user, or list could be undefined?

Reply:
List, yes. I could/should add some help error messages :)

Reply:
or more defensive in the case that it's undefined? i don't know what happens when a snippet has an error, do you?

something like 
```let list = (mablInputs && mablInputs.variables && mablInputs.variables.user && mablInputs.variables.user.list) || ""; ```

maybe you don't need to be _that_ defensive, not sure.

is list a string here? you could default to an empty string `""` if list is undefined.